package.json

```diff
@@ -1,39 +1,57 @@
 {
 	"name": "@financial-times/o-autocomplete",
 	"version": "0.0.0",
-	"description": "autocomplete",
+	"description": "An origami component for autocomplete inputs",
 	"keywords": [
-		"autocomplete"
+		"autocomplete",
+		"typeahead"
 	],
 	"homepage": "https://registry.origami.ft.com/components/o-autocomplete",
 	"bugs": {
-		"url": "https://github.com/Financial-Times/o-autocomplete/issues"
+		"url": "https://github.com/Financial-Times/o-autocomplete/issues",
+		"email": "origami.support@ft.com",
+		"slack": "#origami-support"
 	},
 	"repository": {
 		"type": "git",
 		"url": "git+https://github.com/Financial-Times/o-autocomplete.git"
 	},
 	"license": "MIT",
-	"author": "",
 	"type": "module",
+	"browser": "main.js",
 	"devDependencies": {
+		"@testing-library/dom": "^7.31.2",
+		"@testing-library/user-event": "^13.1.9",
+		"chai": "^4.3.4",
 		"eslint": "^7.26.0",
 		"eslint-config-origami-component": "^2.1.1",
 		"remark": "^13.0.0",
 		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^2.0.0-beta.3",
+		"remark-preset-lint-origami-component": "^2.0.0",
 		"stylelint": "^13.13.1",
 		"stylelint-config-origami-component": "^1.0.4"
 	},
 	"scripts": {
 		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@prerelease\" install && npm_config_yes=true npx \"origami-build-tools@prerelease\" demo"
+		"build": "npm_config_yes=true npx \"origami-build-tools@^11\" install && npm_config_yes=true npx \"origami-build-tools@^11\" demo",
+		"test": "obt t"
 	},
 	"engines": {
 		"npm": ">=7"
 	},
-	"browser": "main.js",
 	"peerDependencies": {
-		"@financial-times/o-brand": "^4.0.0-beta.0"
+		"@financial-times/o-brand": "prerelease",
+		"@financial-times/o-colors": "prerelease",
+		"@financial-times/o-forms": "prerelease",
+		"@financial-times/o-grid": "prerelease",
+		"@financial-times/o-icons": "prerelease",
+		"@financial-times/o-loading": "prerelease",
+		"@financial-times/o-normalise": "prerelease",
+		"@financial-times/o-spacing": "prerelease",
+		"@financial-times/o-typography": "prerelease",
+		"@financial-times/o-visual-effects": "prerelease"
```

Comment:
Need to update to latest stable versions

Reply:
Done in [`18d324b` (#2)](https://github.com/Financial-Times/o-autocomplete/pull/2/commits/18d324b50e01dbd59236f28b1b31e1b29bb3bb61)